examples/redis-operator.md

```diff
@@ -0,0 +1,109 @@
+# Redis-pmem operator
+This a complete example to deploy a Redis cluster through the [redis-operator](https://github.com/spotahome/redis-operator) using QEMU-emulated persistent memory devices.
+
+## PMEM-CSI installation
+The steps to install PMEM-CSI within a K8s cluster with one master and three workers nodes using a fix version of [Clear Linux OS](https://clearlinux.org/) for the nodes, so the following steps can be reproduced, are listed below:
```

Comment:
Replace with this text:  
[FYI I'm not sure if "fixed" is correct in this context. If you mean a specific version # of CL, then replace "fixed" with "specific" and list out the version #.]
The steps below describe how to install PMEM-CSI within a K8S cluster that contains one master and three worker nodes which use a fixed version of [Clear Linux OS](https://clearlinux.org/) for the nodes.

Reply:
I'm using 'specific' instead of 'fixed' now.